src/dlgprefreplaygain.cpp

```diff
@@ -26,24 +35,31 @@ DlgPrefReplayGain::~DlgPrefReplayGain() {
 }
 
 void DlgPrefReplayGain::loadSettings() {
-    if(config->getValueString(ConfigKey(kConfigKey,"ReplayGainEnabled"))==QString("")) {
-        slotResetToDefaults();
-    } else {
-        int iReplayGainBoost =
-                config->getValueString(ConfigKey(kConfigKey, "InitialReplayGainBoost")).toInt();
-        SliderBoost->setValue(iReplayGainBoost);
-        lcddB->display(iReplayGainBoost);
-
-        bool gainEnabled =
-                config->getValueString(ConfigKey(kConfigKey, "ReplayGainEnabled")).toInt() == 1;
-        EnableGain->setChecked(gainEnabled);
-
-        bool analyserEnabled =
-                config->getValueString(ConfigKey(kConfigKey, "ReplayGainAnalyserEnabled")).toInt();
-        EnableAnalyser->setChecked(analyserEnabled);
-    }
+    int iReplayGainBoost = config->getValueString(
+            ConfigKey(kConfigKey, "InitialReplayGainBoost"), "0").toInt();
+    SliderReplayGainBoost->setValue(iReplayGainBoost);
+    LabelCurrentReplayGainBoost->setText(
+            QString::fromUtf8("%1 dB (\xE2\x8C\x80 %2 dB)").arg(
+                  iReplayGainBoost, iReplayGainBoost - 14)); // e.g "0 dB ("Diameter" -14 dB)"
```

Comment:
Maybe I don't either -- let me try:
0dB replay gain adjustment (-14dB overall boost) 

Since replaygain tries to adjust every track to -14dB "loudness" (i.e. using a filter that only passes frequencies humans perceive as loud) then setting the boost to 0dB means that the track will be adjusted to -14dB.